pkg/xds/generator/zoneproxy/generator.go

```diff
@@ -172,3 +172,45 @@ func AddFilterChains(
 
 	return servicesAcc.Services()
 }
+
+const blackHoleClusterName = "kuma:blackhole"
```

Comment:
This should like go to `pkg/xds/envoy/names/resource_names.go`.

Also in light of: https://github.com/kumahq/kuma/issues/8469 and the related issues. We should not introduce more names with `:`. @Automaat created a MADR for listeners: https://github.com/kumahq/kuma/blob/master/docs/madr/decisions/036-internal-listeners.md . I think we should have a similar one for cluster names.
I think we should have `_blackhole` here which is good enough.

Reply:
Agreed, we definitely need a unified naming convension. I'll take a look at #8469
I introduced the colon because we had done so for previous listeners like passthrough ones and a colon simply prevent duplication to user defined services.